src/RowSource.js

```diff
@@ -1,15 +1,15 @@
 export default {
   beginDrag(props) {
     return {
-      rIndex: props.rIndex
```

Comment:
Using row index based on the array was misleading because as soon as the array changes due to a splice action all the index were outdated.